task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +272,16 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    return ccn.toString().split('').reverse().map((value, ind) => { 
+        if (ind % 2 !== 0) {
+            
+            if(2 * value > 9){
```

Comment:
Need to observe indentation (for all functions)
https://javascript.info/coding-style